templates/viz-codelab/src/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "dscc-gen-codelab",
+  "organization": "MyOrg",
+  "description": "My first visualization package.",
+  "logoUrl": "https://logo",
```

Comment:
Can you make this a static link to a graph icon or something? Once we have a gallery, it'd be nice if the codelab didn't give them something with broken images.